CRITICAL - 2025-06-12 10:56:10 --> ErrorException: Undefined array key "idturno"
[Method: GET, Route: admin/editarDirectivo/205]
in APPPATH\Views\editar_directivo.php on line 144.
 1 APPPATH\Views\editar_directivo.php(144): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "idturno"', 'C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Views\\editar_directivo.php', 144)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Views\\editar_directivo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('editar_directivo', [], true)
 5 APPPATH\Controllers\AdminController.php(120): view('editar_directivo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->editarDirectivo('205')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-12 11:01:20 --> mysqli_sql_exception: Unknown column 'curso' in 'field list' in C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\juanxiomaram2024\tesina2025\app\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
#9 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
#10 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\juanxiomaram2024\tesina2025\public\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-12 11:01:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'curso' in 'field list'
[Method: POST, Route: directivo/agregarUsuario]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (:usuario:, :email.1:, :password:, :curso:, :idrol:, :token:, :idcolegio:, :fecha_registro:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:01:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'curso' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$J.qU9uWbN6v7odC9BirHxeQOtdpLLMjnOWNWQ8o4OIc0w06rGta7e\', \'sexto\', \'3\', \'7f5a546a154102ed0e9b744a88bb4da7fdf1142f235dff986f0d717cee6fd22e1633ce8019df936fb1feb469698fa22ae56d\', \'4\', \'2025-06-12 11:01:19\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$J.qU9uWbN6v7odC9BirHxeQOtdpLLMjnOWNWQ8o4OIc0w06rGta7e\', \'sexto\', \'3\', \'7f5a546a154102ed0e9b744a88bb4da7fdf1142f235dff986f0d717cee6fd22e1633ce8019df936fb1feb469698fa22ae56d\', \'4\', \'2025-06-12 11:01:19\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (:usuario:, :email.1:, :password:, :curso:, :idrol:, :token:, :idcolegio:, :fecha_registro:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:01:20 --> [Caused by] mysqli_sql_exception: Unknown column 'curso' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$J.qU9uWbN6v7odC9BirHxeQOtdpLLMjnOWNWQ8o4OIc0w06rGta7e\', \'sexto\', \'3\', \'7f5a546a154102ed0e9b744a88bb4da7fdf1142f235dff986f0d717cee6fd22e1633ce8019df936fb1feb469698fa22ae56d\', \'4\', \'2025-06-12 11:01:19\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$J.qU9uWbN6v7odC9BirHxeQOtdpLLMjnOWNWQ8o4OIc0w06rGta7e\', \'sexto\', \'3\', \'7f5a546a154102ed0e9b744a88bb4da7fdf1142f235dff986f0d717cee6fd22e1633ce8019df936fb1feb469698fa22ae56d\', \'4\', \'2025-06-12 11:01:19\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$J.qU9uWbN6v7odC9BirHxeQOtdpLLMjnOWNWQ8o4OIc0w06rGta7e\', \'sexto\', \'3\', \'7f5a546a154102ed0e9b744a88bb4da7fdf1142f235dff986f0d717cee6fd22e1633ce8019df936fb1feb469698fa22ae56d\', \'4\', \'2025-06-12 11:01:19\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (:usuario:, :email.1:, :password:, :curso:, :idrol:, :token:, :idcolegio:, :fecha_registro:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-12 11:01:55 --> mysqli_sql_exception: Unknown column 'curso' in 'field list' in C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\juanxiomaram2024\tesina2025\app\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
#9 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
#10 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\juanxiomaram2024\tesina2025\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\juanxiomaram2024\tesina2025\public\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-12 11:01:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'curso' in 'field list'
[Method: POST, Route: directivo/agregarUsuario]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (:usuario:, :email.1:, :password:, :curso:, :idrol:, :token:, :idcolegio:, :fecha_registro:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:01:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'curso' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$3g1B/0q8ahmX9WMtpP9Uy.DVcuU8tA1qLL18a7HsJkfmI.9QUv23C\', \'sexto\', \'3\', \'5aba693b93ff86cb7cc8c5ad66871c17cb03842c822d417740be9ee8cefbe94cd93292693275196605517175ba8c3a2fe8ac\', \'4\', \'2025-06-12 11:01:55\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$3g1B/0q8ahmX9WMtpP9Uy.DVcuU8tA1qLL18a7HsJkfmI.9QUv23C\', \'sexto\', \'3\', \'5aba693b93ff86cb7cc8c5ad66871c17cb03842c822d417740be9ee8cefbe94cd93292693275196605517175ba8c3a2fe8ac\', \'4\', \'2025-06-12 11:01:55\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (:usuario:, :email.1:, :password:, :curso:, :idrol:, :token:, :idcolegio:, :fecha_registro:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:01:55 --> [Caused by] mysqli_sql_exception: Unknown column 'curso' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$3g1B/0q8ahmX9WMtpP9Uy.DVcuU8tA1qLL18a7HsJkfmI.9QUv23C\', \'sexto\', \'3\', \'5aba693b93ff86cb7cc8c5ad66871c17cb03842c822d417740be9ee8cefbe94cd93292693275196605517175ba8c3a2fe8ac\', \'4\', \'2025-06-12 11:01:55\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$3g1B/0q8ahmX9WMtpP9Uy.DVcuU8tA1qLL18a7HsJkfmI.9QUv23C\', \'sexto\', \'3\', \'5aba693b93ff86cb7cc8c5ad66871c17cb03842c822d417740be9ee8cefbe94cd93292693275196605517175ba8c3a2fe8ac\', \'4\', \'2025-06-12 11:01:55\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (\'prueba h3\', \'juanjojojolvl1@gmail.com\', \'$2y$10$3g1B/0q8ahmX9WMtpP9Uy.DVcuU8tA1qLL18a7HsJkfmI.9QUv23C\', \'sexto\', \'3\', \'5aba693b93ff86cb7cc8c5ad66871c17cb03842c822d417740be9ee8cefbe94cd93292693275196605517175ba8c3a2fe8ac\', \'4\', \'2025-06-12 11:01:55\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `email`, `password`, `curso`, `idrol`, `token`, `idcolegio`, `fecha_registro`) VALUES (:usuario:, :email.1:, :password:, :curso:, :idrol:, :token:, :idcolegio:, :fecha_registro:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DirectivoController.php(83): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:02:04 --> ErrorException: Undefined array key "curso"
[Method: POST, Route: directivo/agregarUsuario]
in APPPATH\Controllers\DirectivoController.php on line 72.
 1 APPPATH\Controllers\DirectivoController.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "curso"', 'C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Controllers\\DirectivoController.php', 72)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->agregarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:02:35 --> ErrorException: Undefined array key "curso"
[Method: GET, Route: gestionar_usuarios]
in APPPATH\Views\gestionar_usuarios.php on line 216.
 1 APPPATH\Views\gestionar_usuarios.php(216): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "curso"', 'C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Views\\gestionar_usuarios.php', 216)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Views\\gestionar_usuarios.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('gestionar_usuarios', [], true)
 5 APPPATH\Controllers\DirectivoController.php(27): view('gestionar_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->gestionarUsuarios()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:03:19 --> ErrorException: Undefined array key "curso"
[Method: GET, Route: directivo/editarUsuario/206]
in APPPATH\Views\editar_usuario.php on line 152.
 1 APPPATH\Views\editar_usuario.php(152): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "curso"', 'C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Views\\editar_usuario.php', 152)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Views\\editar_usuario.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('editar_usuario', [], true)
 5 APPPATH\Controllers\DirectivoController.php(146): view('editar_usuario', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->editarUsuario('206')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-12 11:03:46 --> ErrorException: Undefined array key "curso"
[Method: POST, Route: directivo/actualizarUsuario]
in APPPATH\Controllers\DirectivoController.php on line 165.
 1 APPPATH\Controllers\DirectivoController.php(165): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "curso"', 'C:\\xampp\\htdocs\\juanxiomaram2024\\tesina2025\\app\\Controllers\\DirectivoController.php', 165)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DirectivoController->actualizarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DirectivoController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
